README.md

```diff
@@ -98,7 +101,7 @@ response = DummyElasticSearchModel.insert!(model.deep_squash_to_store, model.ind
 DummyElasticSearchModel.insert!(model.deep_squash_to_store, model.index_name)
 => raises ElasticsearchModels::Base::CreateError
 
-# Non-Hash value passed in for first argument"" 
+# Non-Hash value passed in for first argument""
```

Comment:
do we need this trailing `""`?

Reply:
Classic whitespace removal showing previously missed typos